megamek/src/megamek/common/EntityFluff.java

```diff
@@ -0,0 +1,290 @@
+/*
```

Comment:
Whitespace is a bit off in this file.

Reply:
I forgot to change the editor settings when I set up a new workspace post-Gradle conversion. I missed this file when I fixed all the others with tabbed indents.